modules/KalturaSupport/KalturaSupport.json

```diff
@@ -91,13 +91,13 @@
 			"components/dualScreen/dualScreenControlBar.js",
 			"components/dualScreen/displays.js",
 			"components/dualScreen/display.js",
-			"components/dualScreen/cuePointsManager.js",
 			"components/dualScreen/dualScreenExternalControlManager.js",
 			"components/dualScreen/imagePlayer.js",
 			"components/dualScreen/videoPlayer.js",
 			"components/dualScreen/videoSync.js",
 			"components/dualScreen/fsm.js",
-			"components/dualScreen/dualScreenStatesConfig.js"
+			"components/dualScreen/dualScreenStatesConfig.js",
+			"resources/webcast/cuePointsManager.js"
```

Comment:
@OrenMe Amir guided me to move the cuePointsManager to 'resources/webcast' so I will be able to use this component both in the dualscreen and the webcastpolls plugins.

He used a dedicated name 'webcast' in 'resources' to indicate that it is used by plugins of webcast